listenbrainz/spotify_updater/spotify_read_listens.py

```diff
@@ -117,12 +118,12 @@ def _convert_spotify_play_to_listen(play, listen_type):
     return listen
 
 
-def make_api_request(user, endpoint, **kwargs):
+def make_api_request(user, spotipy_call, **kwargs):
     """ Make an request to the Spotify API for particular user at specified endpoint with args.
 
     Args:
         user (spotify.Spotify): the user whose plays are to be imported.
-        endpoint (str): the Spotify API endpoint to which the request is to be made
+        spotipy_call (function call): the Spotipy call which makes request to the required API endpoint
```

Comment:
function call is not a valid type. this would take a function, right?

Reply:
done